pkgs/development/tools/kubectx/default.nix

```
@@ -13,20 +11,15 @@ stdenv.mkDerivation rec {
     sha256 = "1b22jk8zl944w5zn3s7ybkkbmzp9519x32pfqwd1malfly7dzf55";
   };
 
-  buildInputs = [ makeWrapper ];
-
-  dontBuild = true;
-  doCheck = false;
+  goPackagePath = "github.com/ahmetb/kubectx";
+  vendorSha256 = "168hfdc2rfwpz2ls607bz5vsm1aw4brhwm8hmbiq1n1l2dn2dj0y";
 
```

if you do the installShellCompleltion:
```suggestion
  nativeBuildInputs = [ installShellFiles ];
```